pkg/volume/csi/constants/constants.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Wrong year

Reply:
Changed. Thanks for the review, @jsafrane!